File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcZoomPages/ZoomPolicy/ZoomPolicyProvider.vb
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace C1ZoomPages
	Public Class ZoomPolicyProvider
		Inherits ZoomPolicyProviderBase
		Public Overrides ReadOnly Property PolicyGroups() As String()
			Get
				Return New String() {"Standard"}
			End Get
		End Property

	End Class
End Namespace

File: Next/TouchToolkit/CS/TouchToolkitExplorer/GcZoomPages/ZoomPolicy/ZoomPolicyProvider.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace TouchToolkitExplorer.C1ZoomPages
{
    public class ZoomPolicyProvider : ZoomPolicyProviderBase
    {
        public override string[] PolicyGroups
        {
            get { return new string[]{"Standard"}; }
        }

    }
}
